man/systemd-journald.service.xml

```diff
@@ -287,7 +284,19 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
         <filename>/var/log/journal</filename> is not available, or
         when <option>Storage=volatile</option> is set in the
         <citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-        configuration file.</para></listitem>
+        configuration file.</para>
+
+	<para>When <filename>systemd-journald</filename> stops writing to a file,
+	it will renamed to <literal><replaceable>original-name</replaceable>@<replaceable>suffix.journal</replaceable></literal>
+	(or <literal><replaceable>original-name</replaceable>@<replaceable>suffix.journal~</replaceable></literal>).
+	Such files are "archived" and will not be written to any more.</para>
+
+	<para>In general, it is safe to copy any journal file (active and archived files).
```

Comment:
what do you mean by "copy"?

Reply:
do you mean "copy at any time"?

Reply:
I changed it to "read or copy". What I mean is that "taking a snapshot of the contents of the file and using that" is OK, but I don't know how to phrase that succinctly.